content/tutorials/atmos-getting-started.md

```diff
@@ -0,0 +1,270 @@
+
+Let's clone it to your local machine:
+
+```bash
+git clone git@github.com:cloudposse/tutorials.git
+```
+
+Now that is on your local machine, let's get into the code that walks us through using `atmos`:
+
+```bash
+cd tutorials/02-atmos
+```
+
+### 2. Build the image for our tutorial
+
+Now that we've got our code located on your local machine, let's look at our tutorial's example directory:
+
+```
+.
+├── Dockerfile
+├── Makefile
+├── README.md
+├── components/
+└── stacks/
+```
+
+Here we've got a `Dockerfile` and a few other things going on. Let's pop open that `Dockerfile` and look at it quickly:
+
+```Dockerfile
+ARG VERSION=0.141.6
+ARG OS=alpine
+ARG CLI_NAME=atmos
+
+FROM cloudposse/geodesic:$VERSION-$OS
+
+# Geodesic message of the Day
+ENV MOTD_URL="https://geodesic.sh/motd"
+
+ENV DOCKER_IMAGE="cloudposse/atmos"
+ENV DOCKER_TAG="latest"
+
+# Geodesic banner
+ENV BANNER="Atmos Tutorial"
+
+# Install terraform.
+RUN apk add -u terraform-0.14@cloudposse
+# Set Terraform 0.14.x as the default `terraform`.
+RUN update-alternatives --set terraform /usr/share/terraform/0.14/bin/terraform
+
+# Install Atmos
+RUN apk add atmos@cloudposse
+
+COPY components/ /components/
+COPY stacks/ /stacks/
+
+WORKDIR /
+```
+
+Few important pieces to point out here:
+
+1. We're using [Geodesic]({{< relref "reference/tools.md#geodesic" >}}) as our base image. This enables us to provide a consistent, reproducible toolbox to invoke `atmos` from.
+1. We're installing the Terraform 0.14 and using that as our default `terraform` executable.
+1. We're installing `atmos` as a simple binary via `apk`. This is because `atmos` is built and distributed as a [Cloud Posse linux package](https://github.com/cloudposse/packages).
+1. We're copying our `components/` and `stacks/` folder into the image.
+
+Overall, a fairly simple and small set of additions on top of standard Geodesic. To get started using this image, first we have to build it. To do so, we could invoke `docker build` manually, but speed things up we've provided a `make` target to simplify that process:
+
+```bash
+# Pull the Cloud Posse build-harness, which provides some additional utilities
+# that our `build` target uses.
+make init
+
+# Build our local Dockerfile onto out local machines as `cloudposse/atmos:latest`
+make build
+```
+
+Once our image is built, we're ready to run it!
+
+### 3. Run the `atmos` image
+
+Now that we've built our image, we want to run it as a new geodesic shell so we can work on our example. Let's run the following command to do that:
+
+```bash
+docker run -it \
+           --rm \
+           --volume $HOME:/localhost \
+           --volume $PWD/stacks:/stacks \
+           --volume $PWD/components:/components \
+           cloudposse/atmos:latest \
+           --login
```

Comment:
@Nuru has updated this so we no longer need to pass login. (although it still works).
```suggestion
           cloudposse/atmos:latest
```

Reply:
Awesome 👍